en/docs/learn/configuring-access-delegation-with-oauth2.md

```diff
@@ -0,0 +1,60 @@
+# Configuring Access Delegation with OAuth 2.0
+
+### Introduction
+
+This tutorial demonstrates how WSO2 Identity Server (WSO2 IS) can be used as an OAuth 2.0 authorization server. It guides you through the OAuth 2.0 application configuration, deployment, and the usage scenario.
+
+In this sample scenario, a user tries to login to a vehicle booking web application called "pickup-dispatch" via a WSO2 Identity Server and recieves an access token to call the API to view the vehicle bookings.
+
+![access-delegation-oauth-diagram](../assets/img/learn/access-delegation-oauth-diagram.png)
+
+### Setting up
+
+1. [Download WSO2 Identity Server](https://wso2.com/identity-and-access-management/).
+
+2. Navigate to `<IS_HOME>/bin` and start the server by executing one of the following commands.
```

Comment:
To start WSO2 Identity Server, navigate to the `<IS_HOME>/bin` directory via command prompt and execute the following command.